contracts/NewfiAdvisor.sol

```diff
@@ -3,22 +3,46 @@ pragma solidity ^0.6.2;
 import "@openzeppelin/contracts-ethereum-package/contracts/token/ERC20/IERC20.sol";
 import "@openzeppelin/contracts-ethereum-package/contracts/token/ERC20/SafeERC20.sol";
 import "@openzeppelin/contracts-ethereum-package/contracts/utils/ReentrancyGuard.sol";
+import "@openzeppelin/contracts-ethereum-package/contracts/Initializable.sol";
+import "./utils/ProxyFactory.sol";
+import "./utils/OwnableUpgradeSafe.sol";
 
-contract NewfiAdvisor is ReentrancyGuardUpgradeSafe {
-    using SafeERC20 for IERC20;
-    using SafeMath for uint;
 
-    event AdvisorOnBoarded(string name, address pool, uint256 stakedAmount);
+contract PoolProxy is Initializable, OwnableUpgradeSafe {
```

Comment:
Should we rename this file to "PoolProxy.sol"?

Reply:
okay right?

Reply:
yea all good.